searx/templates/oscar/result_templates/images.html

```diff
@@ -16,7 +16,7 @@ <h4 class="modal-title">{% if result.engine~".png" in favicons %}{{ draw_favicon
                 {% if result.author %}<span class="photo-author">{{ result.author }}</span><br />{% endif %}
                 {% if result.content %}
                     <p class="result-content">
-                        {{ result.content }}
+                        {{ result.content|striptags }}

```

Comment:
Tags are *added* in `highlight_content`(defined in `searx/utils.py`) and they make sense in the context of textual search. This is not the case for image search however where they are displayed as raw html. This change strips the tags again, it would be more efficient to not have to strip them in the first place. This could be achieved by setting a flag which keeps `highlight_content` from inserting tags or by using another key in the template.